zeekclient/types.py

```diff
@@ -337,24 +337,24 @@ def get(typ, *keys):
 
         # Validate the specified values
         if not instance:
-            raise ValueError('node "{}" requires an instance'.format(name))
+            raise ValueError('node requires an instance')
 
         if not role:
-            raise ValueError('node "{}" requires a role'.format(name))
+            raise ValueError('node requires a role')
 
         try:
             role = ClusterRole.lookup(role)
         except (AttributeError, KeyError) as err:
-            raise ValueError('node "{}" role "{}" is invalid'.format(name, role)) from err
+            raise ValueError('role "{}" is invalid'.format(role)) from err
```

Comment:
I recommend switching everything to f-strings, but that should be a separate PR.